utils/keys.go

```diff
@@ -117,53 +116,47 @@ func ImportKeys(from io.Reader, to []Importer, fallbackRole string, fallbackGun
 				}
 			}
```

Comment:
fixed, with test in TestBlockHeaderPrecedenceGunFromPath. Will move the migration down a level tomorrow morning and this should be good to go 😄